File: Joueur.cs
namespace exercice_trombone
{
    /// <summary>
    /// Joueur de la partie.
    /// </summary>
    internal class Joueur
    {
        /// <summary>
        /// Argent total du joueur.
        /// </summary>
        private int argent;

        /// <summary>
        /// Paie le montant donné si le joueur a assez d'argent.
        /// </summary>
        /// <param name="montant">Le montant à payer.</param>
        /// <returns>True si le montant a été payé, false sinon.</returns>
        public bool Payer(int montant)
        {
            if (argent >= montant)
            {
                argent -= montant;
                return true;
            }

            return false;
        } 
    }
}

File: src/edu/cegepvicto/usine/Joueur.java
package edu.cegepvicto.usine;

/**
 * Classe représentant la personne qui joue au jeu
 */
public class Joueur {

    /**
     * Les fonds disponibles par le joueur
     */
    private int argent;

    /**
     * Le joueur paie le montant indiqué si possible.
     * @param montant le montant à payer
     * @return true si le montant a pu être payé, false sinon.
     */
    public boolean payer(int montant) {
        if(argent >= montant) {
            argent -= montant;
            return true;
        }

        return false;
    }

}
